modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/MysqlSchemaCodegen.java

```diff
@@ -464,8 +461,8 @@ public void processDecimalTypeProperty(CodegenModel model, CodegenProperty prope
         } else {
             Float min = (minimum != null) ? Float.valueOf(minimum) : null;
             Float max = (maximum != null) ? Float.valueOf(maximum) : null;
-            if (exclusiveMinimum == true && min != null) min += 1;
-            if (exclusiveMaximum == true && max != null) max -= 1;
+            if (exclusiveMinimum && min != null) min += 1;
```

Comment:
I used long declaration for readability, but it's ok. Don't like shortcuts in `conditions` because it's hard to read long `if` statements like `if (!isPropValid() && !countErrors() && !$someVar){}`.